src/schema/artwork/context.ts

```diff
@@ -25,12 +26,18 @@ export const ArtworkContextPartnerShowType = create(PartnerShow.type, {
   isTypeOf: ({ context_type }) => context_type === "PartnerShow",
 })
 
+export const ArtworkContextShowType = create(Show.type, {
+  name: "ArtworkContextShow",
+  isTypeOf: ({ context_type }) => context_type === "Show",
+})
+
 export const ArtworkContextType = new GraphQLUnionType({
   name: "ArtworkContext",
   types: [
     ArtworkContextAuctionType,
     ArtworkContextFairType,
     ArtworkContextPartnerShowType,
+    ArtworkContextShowType,
     ArtworkContextSaleType,
```

Comment:
I never really understood why we needed those types, as they are 1:1 copies of the normal types. Perhaps we should remove those from v2?